SIDFactoryII/source/libraries/ghc/filesystem.h

```diff
@@ -171,7 +171,7 @@
 #define LWG_2682_BEHAVIOUR
 //- - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
 // LWG #2395 makes crate_directory/create_directories not emit an error if there is a regular
-// file with that name, it is superceded by P1164R1, so only activate if really needed
```

Comment:
The folder libraries contains dependencies that are copied from the source. Please do not change these files, as it will make upgrading dependencies harder.